frontend/public/proto/game/player.proto

```
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package game;
```

Buf (1.66.1)

[error] Files with package "game" must be within a directory "game" relative to root but were in directory "proto".

(PACKAGE_DIRECTORY_MATCH)